ansible/roles/ml-analytics-service/tasks/main.yml

```diff
@@ -105,6 +105,10 @@
     dest: "{{ config_path }}/config.ini"
     backup: yes
 
+- name: Copy GCP Secrets to JSON file
+  copy: dest="{{config_path}}/{{ml_GCP_secret_json_file}}" content="{{ ml_GCP_Secrets | to_nice_json}}" mode=0644 owner="{{ USER }}"
```

Comment:
@raghupathiguduri can we change the mode to **mode=0400** only the owner will need the read permission,right ?

Reply:
Changed the mode @santhosh-tg